src/client/fuse_ll.cc

```diff
@@ -1263,12 +1374,9 @@ CephFuse::Handle::Handle(Client *c, int fd) :
   se(NULL),
 #if FUSE_VERSION < FUSE_MAKE_VERSION(3, 0)
   ch(NULL),
-  mountpoint(NULL),
+  mountpoint(NULL)
 #endif
-  last_stag(0)
 {
```

Comment:
@lxbsz This is breaking my build with `FUSE_VERSION < FUSE_MAKE_VERSION(3, 0)`:
```
/home/miguel/dev/ceph/ceph/src/client/fuse_ll.cc: In constructor ‘CephFuse::Handle::Handle(Client*, int)’:
/home/miguel/dev/ceph/ceph/src/client/fuse_ll.cc:1379:1: error: expected identifier before ‘{’ token
 1379 | {
      | ^
```

Reply:
Fixed in https://github.com/ceph/ceph/pull/47026. Thanks Luis.